### PR TITLE
Dashboard Basics

### PR DESCRIPTION
**Pull request by Briguy52**
Make sure these boxes are checked before submitting your pull request for review!

- [x] I have written a brief description of what I did
- [x] I have reviewed this code myself
- [ ] I have checked that the TravisCI build passes
- [x] I have removed all console.log() statements
- [ ] There are no new compiler warnings
- [ ] I have tested this on Chrome, Firefox, and Safari - if you're having trouble with Firefox or Safari add import 'whatwg-fetch' to fetch/requestManager.js
- [ ] I have listed .env changes (if applicable)

If this PR is related to UI:

- [x] I have tested this on all screen sizes (iPhones, iPad, Android phones, landscape, portrait)

Testing (if applicable): 

- [ ] I have added unit tests

After submitting, be sure to:

- [ ] Label this PR as 'request to review'

# Done
- [x] Basic layout and styling matches Sketch

- [x] Mobile responsiveness (iPhone, Android, iPad, resized desktop browser) works... 
* `Great` without the navbar
* `Good` with navbar width set to a a vw value (I chose 30)
* `Really bad` for mobile if navbar width is left in px (takes up the entire screen)

# Ask David/Andy/Google
- [ ] How to write tests

- [x] How to change color of user status (accepted/waitlist/etc.) based on prop value

- [x] How to extract content (won't let me create class variables)

# Todo
- [ ] Navbar- collapse on mobile? 

- [ ] Tinker with spacing/padding between sections

- [ ] Add actual content in place of lorem ipsum

### CODE REVIEW

**Comment by Briguy52** [comment]
Note: I'll fix the lint errors about double quotes/really long lines/etc. in next commit

File: src/components/Dashboard/Dashboard.scss [hunk 1]
```diff
@@ -1,11 +1,88 @@
...
+}
+
 .container {
-  padding-left: $navMenuWidth;
+  padding-top: 50px;
+  // padding-left: $navMenuWidth;
 }
 
+.contentContainer {
+  // width: calc(100vw - $navMenuWidth); // TODO: calculate this programmatically
```
Comment (ownzandy) [inline src/components/Dashboard/Dashboard.scss]: why is this commented out?

File: src/components/Dashboard/Dashboard.scss [hunk 2]
```diff
@@ -1,11 +1,88 @@
...
-  font-size: 30px;
-}
\ No newline at end of file
+  color: $standardText;
+  font-size: 12px;
+  margin: 3vh 0 0 0;
+}
+
+@media (min-width: $tabletBreakpoint) {
+  // .contentContainer {
```
Comment (ownzandy) [inline src/components/Dashboard/Dashboard.scss]: same as above

**Comment by ownzandy** [comment]
lol @ TERRIFIC text hover color change effects

**Comment by ownzandy** [comment]
@ changing color based on props value - define a function that takes in the status and returns a color and do it inline (re: https://github.com/hack-duke/hackduke-admin/blob/master/src/components/RoleCell/RoleCell.js)
@ tests - I can explain them as a group, but they should be pretty easy
@ navbar-stuff - @dy46 should be taking care of that
@ extracted-content - since it's static we don't really have to but you can use a constructor to set state and then access the state using this.state.content (re: https://github.com/hack-duke/hackduke-admin/blob/master/src/components/RoleList/RoleList.js)

**Pull request merged**